e2e/core/login/login-sso/login-sso.e2e.ts

```diff
@@ -39,21 +39,21 @@ describe('Login component - SSO', () => {
         });
 
         it('[C261050] Should be possible login with SSO', () => {
-            settingsPage.setProviderEcmSso(TestConfig.adf.url, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false, true, 'alfresco');
+            settingsPage.setProviderEcmSso(TestConfig.adf.url, TestConfig.adf_acs.hostSso, TestConfig.adf_acs.hostIdentity, false, true, 'alfresco');
             loginSSOPage.clickOnSSOButton();
             loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
         });
 
         it('[C280667] Should be redirect directly to keycloak without show the login page with silent login', () => {
-            settingsPage.setProviderEcmSso(TestConfig.adf.url, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, true, true, 'alfresco');
+            settingsPage.setProviderEcmSso(TestConfig.adf.url, TestConfig.adf_acs.hostSso, TestConfig.adf_acs.hostIdentity, true, true, 'alfresco');
             loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
         });
     });
 
     describe('SSO Login Error for login component', () => {
 
         it('[C299205] Should display the login error message when the SSO identity service is wrongly configured', async() => {
-            await settingsPage.setProviderEcmSso(TestConfig.adf.url, 'http://aps22/auth/realms/alfresco', TestConfig.adf.hostIdentity, false, true, 'alfresco');
+            await settingsPage.setProviderEcmSso(TestConfig.adf.url, 'http://aps22/auth/realms/alfresco', TestConfig.adf_acs.hostIdentity, false, true, 'alfresco');
```

Comment:
contains real http hardcoded?

Reply:
In this case it's a fake url. It doesn't mutter if it has it or not, as for the ones coming from the TestConfig we'll need to improve them and reduce the number of duplicated stuff that we currently have